Source/Core/DolphinLib.vcxproj

```diff
@@ -68,31 +68,29 @@
   <ItemGroup>
     <Text Include="$(BuildInfoTemplate)" />
   </ItemGroup>
-  <UsingTask TaskName="GetProductVersion"
-    TaskFactory="CodeTaskFactory"
-    AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.Core.dll">
+  <ItemGroup>
+    <ClInclude Include="Core\TimePlayed.h" />
+  </ItemGroup>
+  <ItemGroup>
+    <ClCompile Include="Core\TimePlayed.cpp" />
+  </ItemGroup>
```

Comment:
Those two should likely go into `DolphinLib.props` (which you'll have to do by hand; VS won't help you there - and make sure you put them in alphabetical order in their respective group).

Also, for consistency, make sure to add things to `Core\CMakeLists.txt` as well.

The rest of this file should be reverted/removed from the commit.